gp-includes/things/translation.php

```diff
@@ -661,6 +661,23 @@ public function last_modified( $translation_set ) {
 		return $last_modified;
 	}
 
+	/**
+	 * Get IDs of translations by ID of their original and translation set.
+	 *
+	 * @since 2.3.0
+	 *
+	 * @param int $original_id        ID of original.
+	 * @param int $translation_set_id ID of translation set.
+	 * @return array $ids An array of IDs of matching translations.
+	 */
+	public function ids_by_original_and_set( $original_id, $translation_set_id  ) {
+		global $wpdb;
+
```

Comment:
Perhaps a `define()`  or filter should be here to allow a site to disable this functionality (or enable it if it's disabled by default?).

Reply:
Re performance see https://wordpress.slack.com/archives/C02RP4R9F/p1506447601000336